### ISSUE TITLE
Sketch out branch between STAC entries and Records editing

### ISSUE THREAD

**Issue by j08lue** [EOEPCA/data-access#69]
For EOEPCA, we need to provide an interface that allows for editing both OGC records (pycsw) and STAC collections / items (eoAPI and pycsw).

Records and STAC collections are very similar, but records are a lot more flexible - almost all properties are optional - and the resources that records link to may often be arbitrary files, while STAC collections in eoAPI will usually be raster or vector datasets that can also be accessed through a server.

We therefore expect that workflows for users to edit records or STAC can be quite different: in particular for STAC entries, validation and fast feedback of rendering and extension-based information is important, while Records are a lot simpler.

This ticket is to ideate how we in the UI can account for the two potentially different workflows.

**Comment by j08lue** [comment]
Where to go if you want to add / edit a

* Spatio-temporal assets -> STAC
* Documents such as notebooks, workflows, non-geospatial images -> OGC API Records

**Comment by j08lue** [comment]
For the record, we discussed different options for serving both STAC and OGC API Records today among @ricardoduplos, @danielfdsilva, and @emmanuelmathot.

We clarified the history and complementary purpose of both standards, in EOEPCA+ and more generally.

We decided on moving forward with an early branch between the two standards, but giving users some context to make the choice between them, likely based on the type of asset they want to edit / add (see above).

We also discussed possibilities of offering users a more seamless switch between collections from OGC API Records and STAC, including a self-configuring UI that automatically distinguishes between the two based on the `conformsTo` details of a `Catalog` (e.g. [Planetary Computer STAC](https://planetarycomputer.microsoft.com/api/stac/v1)). We decided to keep that kind of merging for later, expecting that the two standards will also continue to converge.
